src/js/libcs/OpDrawing.js

```diff
@@ -470,9 +470,21 @@ eval_helper.drawconic = function(conicMatrix, modifs) {
     var c02 = mat.value[1].value[1].value.real;
     var c01 = mat.value[1].value[2].value.real * 2;
     var c00 = mat.value[2].value[2].value.real;
-    var discr = c20 * c02 * 4 - c11 * c11;
-    var det = discr * c00 + c11 * c01 * c10 -
-        c20 * c01 * c01 - c02 * c10 * c10;
+
+    // The adjoint matrix k## values
+    var k20 = 4 * c00 * c02 - c01 * c01;
+    var k11 = c01 * c10 - 2 * c00 * c11;
+    var k10 = c01 * c11 - 2 * c02 * c10;
+    var k02 = 4 * c00 * c20 - c10 * c10;
+    var k01 = c10 * c11 - 2 * c01 * c20;
+    var k00 = 4 * c02 * c20 - c11 * c11;
+
+    var discr = k00;
+    var det = c02 * k02 + c11 * k11 + c20 * k20 - c00 * k00;
+
+    // conic center
+    var ccx = k10 / k00;
+    var ccy = k01 / k00;
```

Comment:
Opps...I take that back. I see you are now using the conic center `ccx` and `ccy` values later in your own code so those values need to stay 😳 However `det` is used nowhere else and `k` is not affected by the sign change to the `c##` values. Also the only use of `discr` in `if (discr <= 0)` could be replaced with `k00` and thus `var discr = k00;` would not be needed.